diff -aurBb /home/httpd/htdocs/users/kaygorodov/discounts/var/ait_patch_orig/compare29612809962694c5a73ade58e7 /home/httpd/htdocs/users/kaygorodov/discounts/var/ait_patch_modif/compare29612809962694c5a73ade58e7
--- /home/httpd/htdocs/users/kaygorodov/discounts/var/ait_patch_orig/compare29612809962694c5a73ade58e7	2010-08-05 11:17:49 +0300
+++ /home/httpd/htdocs/users/kaygorodov/discounts/var/ait_patch_modif/compare29612809962694c5a73ade58e7	2010-08-05 11:17:49 +0300
@@ -25,6 +25,23 @@
  */
 ?>
 <form action="" id="co-shipping-form">
+
+
+
+
+
+<!--     START AITOC CHECKOUT ATTRIBUTES     -->
+    <?php if ($aFieldList = $this->getCustomFieldList(1)): ?>
+    <fieldset>
+        <ul class="form-list">
+        <?php foreach ($aFieldList as $aField): ?>
+            <?php echo $this->getFieldHtml($aField); ?>
+        <?php endforeach; ?>
+        </ul>                
+    </fieldset>
+    <?php endif; ?>
+<!--     FINISH AITOC CHECKOUT ATTRIBUTES     -->          
+
     <ul class="form-list">
     <?php if ($this->customerHasAddresses()): ?>
        <li class="wide">
@@ -132,5 +149,20 @@
             <input type="checkbox" name="shipping[same_as_billing]" id="shipping:same_as_billing" value="1"<?php if($this->getAddress()->getSameAsBilling()): ?> checked="checked"<?php endif; ?> title="<?php echo $this->__('Use Billing Address') ?>" onclick="shipping.setSameAsBilling(this.checked)" class="checkbox" /><label for="shipping:same_as_billing"><?php echo $this->__('Use Billing Address') ?></label>
         </li>
     </ul>
+
+
+    
+<!--     START AITOC CHECKOUT ATTRIBUTES     -->
+    <?php if ($aFieldList = $this->getCustomFieldList(2)): ?>
+    <fieldset>
+        <ul class="form-list">
+        <?php foreach ($aFieldList as $aField): ?>
+            <?php echo $this->getFieldHtml($aField); ?>
+        <?php endforeach; ?>
+        </ul>                
+    </fieldset>
+    <?php endif; ?>
+<!--     FINISH AITOC CHECKOUT ATTRIBUTES     -->          
+    
     <div class="buttons-set" id="shipping-buttons-container">
         <p class="back-link"><a href="#" onclick="checkout.back(); return false;"><small>&laquo; </small><?php echo $this->__('Back') ?></a></p>
