diff -aurBb /home/httpd/htdocs/trepachko/var/ait_patch_orig/compare84912946736884d2b2718bd3da /home/httpd/htdocs/trepachko/var/ait_patch_modif/compare84912946736884d2b2718bd3da
--- /home/httpd/htdocs/trepachko/var/ait_patch_orig/compare84912946736884d2b2718bd3da	2011-01-10 17:34:48.000000000 +0200
+++ /home/httpd/htdocs/trepachko/var/ait_patch_modif/compare84912946736884d2b2718bd3da	2011-01-10 17:34:48.000000000 +0200
@@ -31,15 +31,64 @@
  * @see Mage_Checkout_Block_Multishipping_Addresses
  */
 ?>
+
+<?php /* START AITOC COMMENTING
 <form id="checkout_multishipping_form" action="<?php echo $this->getPostActionUrl() ?>" method="post">
+FINISH AITOC COMMENTING */ ?>
+
+<!--     START AITOC CHECKOUT ATTRIBUTES     -->
+<form action="<?php echo $this->getPostActionUrl() ?>" method="post" id="co-addresses-form">
+<!--     FINISH AITOC CHECKOUT ATTRIBUTES     -->          
+
 <div class="page-title title-buttons">
     <h1><?php echo $this->__('Ship to Multiple Addresses') ?></h1>
+
+<?php /* START AITOC COMMENTING
     <button data-action="add-new-customer-address" type="button" title="<?php echo $this->__('Enter a New Address') ?>" class="button" onclick="$('add_new_address_flag').value=1; $('checkout_multishipping_form').submit();"><span><span><?php echo $this->__('Enter a New Address') ?></span></span></button>
+FINISH AITOC COMMENTING */ ?>
+
+<!--     START AITOC CHECKOUT ATTRIBUTES     -->
+    <button data-action="add-new-customer-address" type="submit" title="<?php echo $this->__('Enter a New Address') ?>" class="button" onclick="$('add_new_address_flag').value=1"><span><span><?php echo $this->__('Enter a New Address') ?></span></span></button>
+<!--     FINISH AITOC CHECKOUT ATTRIBUTES     -->
+
 </div>
 <?php echo $this->getMessagesBlock()->getGroupedHtml() ?>
 <fieldset class="multiple-checkout">
     <input type="hidden" name="continue" value="0" id="can_continue_flag" />
     <input type="hidden" name="new_address" value="0" id="add_new_address_flag" />
+
+<!--     START AITOC CHECKOUT ATTRIBUTES     -->
+<script type="text/javascript">
+var Addresses = Class.create();
+Addresses.prototype = {
+    initialize: function(formId){
+        this.formId = formId;
+        this.validator = new Validation(this.formId);
+    },
+    
+    save: function(){
+        if (this.validator.validate()) {
+            return true;
+        }
+        else 
+        {
+            return false;
+        }
+    }
+}
+</script>
+
+    <?php if ($aFieldList = $this->getCustomFieldList(1)): ?>
+    <fieldset>
+        <ul class="form-list">
+        <?php foreach ($aFieldList as $aField): ?>
+            <?php echo $this->getFieldHtml($aField); ?>
+        <?php endforeach; ?>
+        </ul>                
+    </fieldset>
+    <?php endif; ?>
+<!--     FINISH AITOC CHECKOUT ATTRIBUTES     -->          
+
     <h2 class="table-caption"><?php echo $this->__('Please select shipping address for applicable items') ?></h2>
     <table class="data-table" id="multiship-addresses-table">
         <col />
@@ -70,10 +119,37 @@
         <?php endforeach; ?>
         </tbody>
     </table>
+
+<!--     START AITOC CHECKOUT ATTRIBUTES     -->
+    <?php if ($aFieldList = $this->getCustomFieldList(2)): ?>
+    <fieldset>
+        <ul class="form-list">
+        <?php foreach ($aFieldList as $aField): ?>
+            <?php echo $this->getFieldHtml($aField); ?>
+        <?php endforeach; ?>
+        </ul>                
+    </fieldset>
+    <?php endif; ?>
+ 
+<script type="text/javascript">
+//<![CDATA[
+    var addresses = new Addresses('co-addresses-form');
+//]]>
+</script>
+<!--     FINISH AITOC CHECKOUT ATTRIBUTES     -->         
+
     <script type="text/javascript">decorateTable('multiship-addresses-table')</script>
     <div class="buttons-set">
         <p class="back-link"><a href="<?php echo $this->getBackUrl() ?>"><small>&laquo; </small><?php echo $this->__('Back to Shopping Cart') ?></a></p>
+
+<?php /* START AITOC COMMENTING    
         <button type="submit" data-action="checkout-continue-shipping" class="button<?php if ($this->isContinueDisabled()):?> disabled<?php endif; ?>" onclick="$('can_continue_flag').value=1"<?php if ($this->isContinueDisabled()):?> disabled="disabled"<?php endif; ?>><span><span><?php echo $this->__('Continue to Shipping Information') ?></span></span></button>
+FINISH AITOC COMMENTING */ ?>
+
+<!--     START AITOC CHECKOUT ATTRIBUTES     -->
+        <button data-action="checkout-continue-shipping" type="submit" class="button<?php if ($this->isContinueDisabled()):?> disabled<?php endif; ?>" onclick="if (addresses.save()) {$('can_continue_flag').value=1} else {return false;}"<?php if ($this->isContinueDisabled()):?> disabled="disabled"<?php endif; ?>><span><span><?php echo $this->__('Continue to Shipping Information') ?></span></span></button>
+<!--     FINISH AITOC CHECKOUT ATTRIBUTES     -->
+
     </div>
 </fieldset>
 </form>
