diff -aurBb /home/httpd/htdocs/users/martinkevich/ee-11110/var/ait_patch_orig/compare16213209225824ebbadd63a173 /home/httpd/htdocs/users/martinkevich/ee-11110/var/ait_patch_modif/compare16213209225824ebbadd63a173
--- /home/httpd/htdocs/users/martinkevich/ee-11110/var/ait_patch_orig/compare16213209225824ebbadd63a173	2011-11-10 13:56:22 +0300
+++ /home/httpd/htdocs/users/martinkevich/ee-11110/var/ait_patch_modif/compare16213209225824ebbadd63a173	2011-11-10 13:56:22 +0300
@@ -30,12 +30,66 @@
     var checkQuoteBaseGrandTotal = quoteBaseGrandTotal;
     var payment = new Payment('co-payment-form', '<?php echo $this->getUrl('checkout/onepage/savePayment') ?>');
     var lastPrice;
+
+
+
+    payment.init = function(){
+       //this.beforeInit();
+        var elements = Form.getElements(this.form);
+        if ($(this.form)) {
+            $(this.form).observe('submit', function(event){this.save();Event.stop(event);}.bind(this));
+        }
+        var method = null;
+        for (var i=0; i<elements.length; i++) {
+            if ((elements[i].name=='payment[method]')||(elements[i].name.indexOf('aitoc_checkout_')!=-1)) {
+                if (elements[i].checked) {
+                    method = elements[i].value;
+                }
+            } else {
+                elements[i].disabled = true;
+            }
+            elements[i].setAttribute('autocomplete','off');
+        }
+        if (method) this.switchMethod(method);
+        //this.afterInit();
+    }
+
 //]]>
 </script>
 <form action="" id="co-payment-form">
+
+
+
+<!--     START AITOC CHECKOUT ATTRIBUTES     -->
+    <?php if ($aFieldList = $this->getCustomFieldList(1)): ?>
+    <fieldset>
+        <ul class="form-list">
+        <?php foreach ($aFieldList as $aField): ?>
+            <?php echo $this->getFieldHtml($aField); ?>
+        <?php endforeach; ?>
+        </ul>                
+    </fieldset>
+    <?php endif; ?>
+<!--     FINISH AITOC CHECKOUT ATTRIBUTES     -->          
+
     <fieldset>
         <?php echo $this->getChildHtml('methods') ?>
     </fieldset>
+
+
+   
+<!--     START AITOC CHECKOUT ATTRIBUTES     -->
+    <?php if ($aFieldList = $this->getCustomFieldList(2)): ?>
+    <fieldset>
+        <ul class="form-list">
+        <?php foreach ($aFieldList as $aField): ?>
+            <?php echo $this->getFieldHtml($aField); ?>
+        <?php endforeach; ?>
+        </ul>                
+    </fieldset>
+    <?php endif; ?>
+<!--     FINISH AITOC CHECKOUT ATTRIBUTES     -->          
+
 </form>
 <div class="tool-tip" id="payment-tool-tip" style="display:none;">
     <div class="btn-close"><a href="#" id="payment-tool-tip-close" title="<?php echo $this->__('Close') ?>"><?php echo $this->__('Close') ?></a></div>
