diff -aurBb /home/httpd/htdocs/enterprise/vasko-ee11100/var/ait_patch_orig/app/design/frontend/enterprise/default/template/sales/order/info.phtml /home/httpd/htdocs/enterprise/vasko-ee11100/var/ait_patch_modif/app/design/frontend/enterprise/default/template/sales/order/info.phtml
--- /home/httpd/htdocs/enterprise/vasko-ee11100/var/ait_patch_orig/app/design/frontend/enterprise/default/template/sales/order/info.phtml   2011-06-07 17:47:53.000000000 +0000
+++ /home/httpd/htdocs/enterprise/vasko-ee11100/var/ait_patch_modif/app/design/frontend/enterprise/default/template/sales/order/info.phtml  2011-10-26 12:22:50.000000000 +0000
@@ -84,4 +84,33 @@
         </div>
     </div>
 </div>
+
+<!--     START AITOC CHECKOUT ATTRIBUTES     -->
+<?php if ($aCustomAtrrList = $this->getOrderCustomData()) : ?>
+    <!--Account Information-->
+<div class="col2-set">
+        <div class="info-box">
+
+            <h2 class="box-title"><?php echo Mage::getStoreConfig('aitcheckoutfields/common_settings/aitcheckoutfields_additionalblock_label', $this->getStoreId()); ?></h2>
+
+            <div class="box-content">
+
+                <table cellspacing="0" class="form-list" width="100%">
+                
+    <?php foreach ($aCustomAtrrList as $aItem): if(!empty($aItem['value'])){ ?>
+                <tr>
+                    <td class="label" width="50%"><?php echo $aItem['label']; ?></td>
+                    <td>&nbsp;</td>
+                    <td><label style="text-align: left;"><?php echo $aItem['value'] ?></label></td>
+                </tr>
+    <?php } endforeach; ?>
+                
+                </table>
+</div>
+</div>
+</div>
+
+<?php endif; ?>
+<!--     FINISH AITOC CHECKOUT ATTRIBUTES     -->
+
 </div>
Only in /home/httpd/htdocs/enterprise/vasko-ee11100/var/ait_patch_orig/: compare91913196318234ea7fbcfbd279
