diff -aurBb /home/httpd/htdocs/users/kaygorodov/discounts/var/ait_patch_orig/compare33812809962694c5a73adc64e4 /home/httpd/htdocs/users/kaygorodov/discounts/var/ait_patch_modif/compare33812809962694c5a73adc64e4
--- /home/httpd/htdocs/users/kaygorodov/discounts/var/ait_patch_orig/compare33812809962694c5a73adc64e4	2010-08-05 11:17:49 +0300
+++ /home/httpd/htdocs/users/kaygorodov/discounts/var/ait_patch_modif/compare33812809962694c5a73adc64e4	2010-08-05 11:17:49 +0300
@@ -25,6 +25,22 @@
  */
 ?>
 <form id="co-billing-form" action="">
+
+
+
+<!--     START AITOC CHECKOUT ATTRIBUTES     -->
+    <?php if ($aFieldList = $this->getCustomFieldList(1)): ?>
+    <fieldset>
+        <ul class="form-list">
+        <?php foreach ($aFieldList as $aField): ?>
+            <?php echo $this->getFieldHtml($aField); ?>
+        <?php endforeach; ?>
+        </ul>                
+    </fieldset>
+    <?php endif; ?>
+<!--     FINISH AITOC CHECKOUT ATTRIBUTES     -->          
+
+
 <fieldset>
     <ul class="form-list">
     <?php if ($this->customerHasAddresses()): ?>
@@ -145,7 +161,11 @@
                 </li>
         <?php endif ?>
 
+<?php /* START AITOC COMMENTING 
                 <li class="fields" id="register-customer-password">
+FINISH AITOC COMMENTING */ ?>
+            <li id="register-customer-password"><ul>
+                <li class="fields">
                     <div class="field">
                         <label for="billing:customer_password" class="required"><em>*</em><?php echo $this->__('Password') ?></label>
                         <div class="input-box">
@@ -157,6 +179,14 @@
                         </div>
                     </div>
                 </li>
+                <!--     START AITOC CHECKOUT ATTRIBUTES     -->
+                <?php if ($aFieldList = $this->getRegCustomFieldList()): ?>
+                    <?php foreach ($aFieldList as $aField): ?>
+                        <?php echo $this->getFieldHtml($aField); ?>
+                    <?php endforeach; ?>
+                <?php endif; ?>
+                <!--     FINISH AITOC CHECKOUT ATTRIBUTES     --> 
+            </ul></li>
                 <?php endif; ?>
                 <?php if ($this->isCustomerLoggedIn() && $this->customerHasAddresses()):?>
                     <li class="control">
@@ -178,12 +206,36 @@
         </li>
     <?php endif; ?>
     </ul>
+
+
+    
+<!--     START AITOC CHECKOUT ATTRIBUTES     -->
+    <?php if ($aFieldList = $this->getCustomFieldList(2)): ?>
+    <fieldset>
+        <ul class="form-list">
+        <?php foreach ($aFieldList as $aField): ?>
+            <?php echo $this->getFieldHtml($aField); ?>
+        <?php endforeach; ?>
+        </ul>                
+    </fieldset>
+    <?php endif; ?><!--     FINISH AITOC CHECKOUT ATTRIBUTES     -->                
+
+    
+    
     <?php if (!$this->canShip()): ?>
         <input type="hidden" name="billing[use_for_shipping]" value="1" />
     <?php endif; ?>
     <div class="buttons-set" id="billing-buttons-container">
         <p class="required"><?php echo $this->__('* Required Fields') ?></p>
+
+
+        
+<?php /* START AITOC COMMENTING    
         <button type="button" title="<?php echo $this->__('Continue') ?>" class="button" onclick="billing.save()"><span><span><?php echo $this->__('Continue') ?></span></span></button>
+
+
+FINISH AITOC COMMENTING */ ?>        
+        <button type="button" title="<?php echo $this->__('Continue') ?>" class="button" onclick="checkShippReqAttrib(); billing.save()"><span><span><?php echo $this->__('Continue') ?></span></span></button>
         <span class="please-wait" id="billing-please-wait" style="display:none;">
             <img src="<?php echo $this->getSkinUrl('images/opc-ajax-loader.gif') ?>" alt="<?php echo $this->__('Loading next step...') ?>" title="<?php echo $this->__('Loading next step...') ?>" class="v-middle" /> <?php echo $this->__('Loading next step...') ?>
         </span>
@@ -199,5 +251,20 @@
     $('billing-address-select') && billing.newAddress(!$('billing-address-select').value);
 
     var billingRegionUpdater = new RegionUpdater('billing:country_id', 'billing:region', 'billing:region_id', countryRegions, undefined, 'billing:postcode');
+
+
+    
+//     START AITOC CHECKOUT ATTRIBUTES
+   function checkShippReqAttrib()
+   {
+       var bHashRequered = '<?php echo $this->checkStepHasRequired(); ?>';
+       if (!bHashRequered) return true;
+       if ($('billing:use_for_shipping_yes')) $('billing:use_for_shipping_yes').checked = false;
+       if ($('billing:use_for_shipping_no')) $('billing:use_for_shipping_no').checked = true;
+       if ($('billing:same_as_billing')) $('shipping:same_as_billing').checked = false;
+   }
+//     FINISH AITOC CHECKOUT ATTRIBUTES
+    
+    
 //]]>
 </script>
