diff -aurBb /home/httpd/htdocs/trepachko/var/ait_patch_orig/compare43112946736884d2b2718ced1b /home/httpd/htdocs/trepachko/var/ait_patch_modif/compare43112946736884d2b2718ced1b
--- /home/httpd/htdocs/trepachko/var/ait_patch_orig/compare43112946736884d2b2718ced1b	2011-01-10 17:34:48.000000000 +0200
+++ /home/httpd/htdocs/trepachko/var/ait_patch_modif/compare43112946736884d2b2718ced1b	2011-01-10 17:34:48.000000000 +0200
@@ -37,6 +37,39 @@
 <?php echo $this->getMessagesBlock()->getGroupedHtml() ?>
 <form action="<?php echo $this->getPostActionUrl() ?>" method="post" id="shipping_method_form">
 <fieldset class="multiple-checkout">
+
+<!--     START AITOC CHECKOUT ATTRIBUTES     -->
+<script type="text/javascript">
+var Addresses = Class.create();
+Addresses.prototype = {
+    initialize: function(formId){
+        this.formId = formId;
+        this.validator = new Validation(this.formId);
+    },
+    
+    save: function(){
+        if (this.validator.validate()) {
+            return true;
+        }
+        else 
+        {
+            return false;
+        }
+    }
+}
+</script>
+
+    <?php if ($aFieldList = $this->getCustomFieldList(1)): ?>
+    <fieldset>
+        <ul class="form-list">
+        <?php foreach ($aFieldList as $aField): ?>
+            <?php echo $this->getFieldHtml($aField); ?>
+        <?php endforeach; ?>
+        </ul>                
+    </fieldset>
+    <?php endif; ?>
+<!--     FINISH AITOC CHECKOUT ATTRIBUTES     -->          
+
     <?php foreach ($this->getAddresses() as $_index => $_address): ?>
     <h2 class="section-title"><?php echo $this->__('Address %s of %s', ($_index+1), $this->getAddressCount()) ?></h2>
     <div class="col2-set">
@@ -112,9 +145,36 @@
     </div>
     <?php endforeach; ?>
     <?php echo $this->getChildHtml('checkout_billing_items') ?>
+
+<!--     START AITOC CHECKOUT ATTRIBUTES     -->
+    <?php if ($aFieldList = $this->getCustomFieldList(2)): ?>
+    <fieldset>
+        <ul class="form-list">
+        <?php foreach ($aFieldList as $aField): ?>
+            <?php echo $this->getFieldHtml($aField); ?>
+        <?php endforeach; ?>
+        </ul>                
+    </fieldset>
+    <?php endif; ?>
+ 
+<script type="text/javascript">
+//<![CDATA[
+    var addresses = new Addresses('shipping_method_form');
+//]]>
+</script>
+<!--     FINISH AITOC CHECKOUT ATTRIBUTES     -->
+
     <div class="buttons-set">
         <p class="back-link"><a href="<?php echo $this->getBackUrl() ?>"><small>&laquo; </small><?php echo $this->__('Back to Select Addresses') ?></a></p>
+
+<?php /* START AITOC COMMENTING
         <button  class="button" type="submit"><span><span><?php echo $this->__('Continue to Billing Information') ?></span></span></button>
+FINISH AITOC COMMENTING */ ?>
+
+<!--     START AITOC CHECKOUT ATTRIBUTES     -->
+        <button  class="button" type="submit" onclick="if (!addresses.save()) return false;"><span><span><?php echo $this->__('Continue to Billing Information') ?></span></span></button>
+<!--     FINISH AITOC CHECKOUT ATTRIBUTES     -->
+
     </div>
 </fieldset>
 </form>
