diff -aurBb /home/httpd/htdocs/trepachko/var/ait_patch_orig/compare49312946736884d2b271897271 /home/httpd/htdocs/trepachko/var/ait_patch_modif/compare49312946736884d2b271897271
--- /home/httpd/htdocs/trepachko/var/ait_patch_orig/compare49312946736884d2b271897271	2011-01-10 17:34:48.000000000 +0200
+++ /home/httpd/htdocs/trepachko/var/ait_patch_modif/compare49312946736884d2b271897271	2011-01-10 17:34:48.000000000 +0200
@@ -25,6 +25,25 @@
  */
 ?>
 <form action="" id="co-shipping-form">
+
+
+
+
+
+
+
+<!--     START AITOC CHECKOUT ATTRIBUTES     -->
+    <?php if ($aFieldList = $this->getCustomFieldList(1)): ?>
+    <fieldset>
+        <ul class="form-list">
+        <?php foreach ($aFieldList as $aField): ?>
+            <?php echo $this->getFieldHtml($aField); ?>
+        <?php endforeach; ?>
+        </ul>                
+    </fieldset>
+    <?php endif; ?>
+<!--     FINISH AITOC CHECKOUT ATTRIBUTES     -->          
+
     <ul class="form-list">
     <?php if ($this->customerHasAddresses()): ?>
        <li class="wide">
@@ -131,6 +150,23 @@
             <input type="checkbox" name="shipping[same_as_billing]" id="shipping:same_as_billing" value="1"<?php if($this->getAddress()->getSameAsBilling()): ?> checked="checked"<?php endif; ?> title="<?php echo $this->__('Use Billing Address') ?>" onclick="shipping.setSameAsBilling(this.checked)" class="checkbox" /><label for="shipping:same_as_billing"><?php echo $this->__('Use Billing Address') ?></label>
         </li>
     </ul>
+
+
+
+
+    
+<!--     START AITOC CHECKOUT ATTRIBUTES     -->
+    <?php if ($aFieldList = $this->getCustomFieldList(2)): ?>
+    <fieldset>
+        <ul class="form-list">
+        <?php foreach ($aFieldList as $aField): ?>
+            <?php echo $this->getFieldHtml($aField); ?>
+        <?php endforeach; ?>
+        </ul>                
+    </fieldset>
+    <?php endif; ?>
+<!--     FINISH AITOC CHECKOUT ATTRIBUTES     -->          
+    
     <div class="buttons-set" id="shipping-buttons-container">
         <p class="required"><?php echo $this->__('* Required Fields') ?></p>
         <p class="back-link"><a href="#" onclick="checkout.back(); return false;"><small>&laquo; </small><?php echo $this->__('Back') ?></a></p>
