diff -aurBb /home/httpd/htdocs/trepachko/var/ait_patch_orig/compare35912946736884d2b2718a8bb2 /home/httpd/htdocs/trepachko/var/ait_patch_modif/compare35912946736884d2b2718a8bb2
--- /home/httpd/htdocs/trepachko/var/ait_patch_orig/compare35912946736884d2b2718a8bb2	2011-01-10 17:34:48.000000000 +0200
+++ /home/httpd/htdocs/trepachko/var/ait_patch_modif/compare35912946736884d2b2718a8bb2	2011-01-10 17:34:48.000000000 +0200
@@ -24,6 +24,42 @@
  * @license     http://www.magentocommerce.com/license/enterprise-edition
  */
 ?>
+
+
+
+<!--     START AITOC CHECKOUT ATTRIBUTES     -->
+
+<?php if ($aCustomAtrrList = $this->getOrderCustomData()) : ?>
+    <!--Account Information-->
+    
+<div class="col2-set order-info-box">
+        <div class="box">
+
+            <div class="box-title">
+                <h2><?php echo Mage::getStoreConfig('aitcheckoutfields/common_settings/aitcheckoutfields_additionalblock_label', $this->getStoreId()); ?></h2>
+            </div>
+
+            <div class="box-content">
+
+                <table cellspacing="0" class="form-list" width="100%">
+                
+    <?php foreach ($aCustomAtrrList as $aItem): if($aItem['value']){ ?>
+                <tr>
+                    <td class="label" width="30%"><?php echo $aItem['label']; ?></td>
+                    <td>&nbsp;</td>
+                    <td><label><?php echo $aItem['value'] ?></label></td>
+                </tr>
+    <?php } endforeach; ?>
+                
+                </table>
+</div>
+</div>
+</div>
+
+<?php endif; ?>
+
+<!--     FINISH AITOC CHECKOUT ATTRIBUTES     -->
+
 <div class="order-items order-details">
     <?php if ($this->helper('giftmessage/message')->getIsMessagesAvailable('items', $this->getOrder())): ?>
     <script type="text/javascript">
