diff -aurBb /home/httpd/htdocs/trepachko/var/ait_patch_orig/compare52412946736884d2b2718c8f6c /home/httpd/htdocs/trepachko/var/ait_patch_modif/compare52412946736884d2b2718c8f6c
--- /home/httpd/htdocs/trepachko/var/ait_patch_orig/compare52412946736884d2b2718c8f6c	2011-01-10 17:34:48.000000000 +0200
+++ /home/httpd/htdocs/trepachko/var/ait_patch_modif/compare52412946736884d2b2718c8f6c	2011-01-10 17:34:48.000000000 +0200
@@ -28,8 +28,49 @@
     <h1><?php echo $this->__('Review Order') ?></h1>
 </div>
 <?php echo $this->getMessagesBlock()->getGroupedHtml() ?>
+
+<?php /* START AITOC COMMENTING
 <form action="<?php echo $this->getPostActionUrl() ?>" method="post" onsubmit="return showLoader();">
+FINISH AITOC COMMENTING */ ?>
+
+<!--     START AITOC CHECKOUT ATTRIBUTES     -->
+    <form action="<?php echo $this->getPostActionUrl() ?>" method="post" onsubmit="return showLoader();" id="overview_form">
+<!--     FINISH AITOC CHECKOUT ATTRIBUTES     -->
+
     <?php echo $this->getBlockHtml('formkey'); ?>
 <fieldset class="multiple-checkout">
+
+<!--     START AITOC CHECKOUT ATTRIBUTES     -->
+<script type="text/javascript">
+var Addresses = Class.create();
+Addresses.prototype = {
+    initialize: function(formId){
+        this.formId = formId;
+        this.validator = new Validation(this.formId);
+    },
+    
+    save: function(){
+        if (this.validator.validate()) {
+            return true;
+        }
+        else 
+        {
+            return false;
+        }
+    }
+}
+</script>
+
+    <?php if ($aFieldList = $this->getCustomFieldList(1)): ?>
+    <fieldset>
+        <ul class="form-list">
+        <?php foreach ($aFieldList as $aField): ?>
+            <?php echo $this->getFieldHtml($aField); ?>
+        <?php endforeach; ?>
+        </ul>                
+    </fieldset>
+    <?php endif; ?>
+<!--     FINISH AITOC CHECKOUT ATTRIBUTES     -->
+
     <div class="col2-set">
         <h2 class="sub-title"><?php echo $this->__('Billing Information') ?></h2>
         <div class="col-1 narrow">
@@ -177,6 +218,25 @@
 
     <div id="checkout-review-submit">
         <?php echo $this->getChildHtml('agreements') ?>
+
+<!--     START AITOC CHECKOUT ATTRIBUTES     -->
+    <?php if ($aFieldList = $this->getCustomFieldList(2)): ?>
+    <fieldset>
+        <ul class="form-list">
+        <?php foreach ($aFieldList as $aField): ?>
+            <?php echo $this->getFieldHtml($aField); ?>
+        <?php endforeach; ?>
+        </ul>                
+    </fieldset>
+    <?php endif; ?>
+ 
+<script type="text/javascript">
+//<![CDATA[
+    var addresses = new Addresses('overview_form');
+//]]>
+</script>
+<!--     FINISH AITOC CHECKOUT ATTRIBUTES     -->
+
         <p class="grand-total"><strong class="price-label"><?php echo $this->__('Grand Total:') ?></strong> <?php echo $this->helper('checkout')->formatPrice($this->getTotal()) ?></p>
         <div class="buttons-set" id="review-buttons-container">
             <button type="submit" class="button" id="review-button"><span><span><?php echo $this->__('Place Order') ?></span></span></button>
@@ -194,6 +254,9 @@
 
     function showLoader()
     {
+
+        if (!addresses.save()) return false; // AITOC CHECKOUT ATTRIBUTES
+
         if (submitted) {
             return false;
         }
