diff -aurBb /home/httpd/htdocs/users/martinkevich/mag1700/var/ait_patch_orig/compare29713355342744f9aa2c25fb9a /home/httpd/htdocs/users/martinkevich/mag1700/var/ait_patch_modif/compare29713355342744f9aa2c25fb9a
--- /home/httpd/htdocs/users/martinkevich/mag1700/var/ait_patch_orig/compare29713355342744f9aa2c25fb9a	2012-04-27 16:44:34 +0300
+++ /home/httpd/htdocs/users/martinkevich/mag1700/var/ait_patch_modif/compare29713355342744f9aa2c25fb9a	2012-04-27 16:44:34 +0300
@@ -40,6 +40,20 @@
     <?php /* Extensions placeholder */ ?>
     <?php echo $this->getChildHtml('customer.form.register.extra')?>
     <form action="<?php echo $this->getPostActionUrl() ?>" method="post" id="form-validate">
+    
+<!--     START AITOC REGISTRATION ATTRIBUTES     -->
+        <?php if($fieldList = $this->getCustomFieldsList(1)){ ?>
+			<div class="fieldset">
+    	        <h2 class="legend"><?php echo Mage::getStoreConfig('aitcheckoutfields/settings/aitcheckoutfields_reg_label_top', $this->getStoreId()) ?></h2>
+    	        <ul class="form-list">
+    	            <?php foreach ($fieldList as $field){
+    	                echo $this->getAttributeHtml($field,'aitreg','register');	
+    	            } ?>
+    	        <ul class="form-list">
+    	    </div>
+        <?php } ?>
+<!--     FINISH AITOC REGISTRATION ATTRIBUTES     -->
+    
         <div class="fieldset">
             <input type="hidden" name="success_url" value="<?php echo $this->getSuccessUrl() ?>" />
             <input type="hidden" name="error_url" value="<?php echo $this->getErrorUrl() ?>" />
@@ -173,6 +187,20 @@
                 <?php echo $this->getChildHtml('form.additional.info'); ?>
             </ul>
         </div>
+        
+<!--     START AITOC REGISTRATION ATTRIBUTES     -->
+        <?php if($fieldList = $this->getCustomFieldsList(2)){ ?>
+			<div class="fieldset">
+    	        <h2 class="legend"><?php echo Mage::getStoreConfig('aitcheckoutfields/settings/aitcheckoutfields_reg_label_bottom', $this->getStoreId()) ?></h2>
+    	        <ul class="form-list">
+    	            <?php foreach ($fieldList as $field){
+    	                echo $this->getAttributeHtml($field,'aitreg','register');	
+    	            } ?>
+    	        <ul class="form-list">
+    	    </div>
+        <?php } ?>
+<!--     FINISH AITOC REGISTRATION ATTRIBUTES     -->
+        
         <div class="buttons-set">
             <p class="required"><?php echo $this->__('* Required Fields') ?></p>
             <p class="back-link"><a href="<?php echo $this->escapeUrl($this->getBackUrl()) ?>" class="back-link"><small>&laquo; </small><?php echo $this->__('Back') ?></a></p>
