diff -aurBb /home/httpd/htdocs/trepachko/var/ait_patch_orig/compare34312946736884d2b27189d039 /home/httpd/htdocs/trepachko/var/ait_patch_modif/compare34312946736884d2b27189d039
--- /home/httpd/htdocs/trepachko/var/ait_patch_orig/compare34312946736884d2b27189d039	2011-01-10 17:34:48.000000000 +0200
+++ /home/httpd/htdocs/trepachko/var/ait_patch_modif/compare34312946736884d2b27189d039	2011-01-10 17:34:48.000000000 +0200
@@ -25,6 +25,23 @@
  */
 ?>
 <form id="co-shipping-method-form" action="">
+
+
+
+
+
+<!--     START AITOC CHECKOUT ATTRIBUTES     -->
+    <?php if ($aFieldList = $this->getCustomFieldList(1)): ?>
+    <fieldset>
+        <ul class="form-list">
+        <?php foreach ($aFieldList as $aField): ?>
+            <?php echo $this->getFieldHtml($aField); ?>
+        <?php endforeach; ?>
+        </ul>                
+    </fieldset>
+    <?php endif; ?>
+<!--     FINISH AITOC CHECKOUT ATTRIBUTES     -->          
+
     <div id="checkout-shipping-method-load">
        <!-- Content loaded dynamically -->
     </div>
@@ -36,6 +53,23 @@
     <div id="onepage-checkout-shipping-method-additional-load">
         <?php echo $this->getChildHtml('additional') ?>
     </div>
+
+
+
+
+    
+<!--     START AITOC CHECKOUT ATTRIBUTES     -->
+    <?php if ($aFieldList = $this->getCustomFieldList(2)): ?>
+    <fieldset>
+        <ul class="form-list">
+        <?php foreach ($aFieldList as $aField): ?>
+            <?php echo $this->getFieldHtml($aField); ?>
+        <?php endforeach; ?>
+        </ul>                
+    </fieldset>
+    <?php endif; ?>
+<!--     FINISH AITOC CHECKOUT ATTRIBUTES     -->          
+    
     <div class="buttons-set" id="shipping-method-buttons-container">
         <p class="back-link"><a href="#" onclick="checkout.back(); return false;"><small>&laquo; </small><?php echo $this->__('Back') ?></a></p>
         <button type="button" class="button" onclick="shippingMethod.save()"><span><span><?php echo $this->__('Continue') ?></span></span></button>
